From 7ff6bb3588487e26af4e03d08e83e5e4984e9ac8 Mon Sep 17 00:00:00 2001
From: Inochi Amaoto <inochiama@gmail.com>
Date: Sun, 12 Oct 2025 09:46:19 +0800
Subject: [PATCH 193/197] FROMLIST: perf vendor events riscv: add T-HEAD C920V2
 JSON support

T-HEAD C920 has a V2 iteration, which supports Sscompmf. The v2
iteration supports the same perf events.

Reuse T-HEAD c900-legacy JSON file for T-HEAD.

Signed-off-by: Inochi Amaoto <inochiama@gmail.com>
Link: https://lore.kernel.org/r/20251012014620.799046-1-inochiama@gmail.com
Signed-off-by: Han Gao <rabenda.cn@gmail.com>
---
 tools/perf/pmu-events/arch/riscv/mapfile.csv | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/perf/pmu-events/arch/riscv/mapfile.csv b/tools/perf/pmu-events/arch/riscv/mapfile.csv
index 3d3a809a5446..9f76c922dec5 100644
--- a/tools/perf/pmu-events/arch/riscv/mapfile.csv
+++ b/tools/perf/pmu-events/arch/riscv/mapfile.csv
@@ -16,5 +16,6 @@
 #MVENDORID-MARCHID-MIMPID,Version,Filename,EventType
 0x489-0x8000000000000007-0x[[:xdigit:]]+,v1,sifive/u74,core
 0x5b7-0x0-0x0,v1,thead/c900-legacy,core
+0x5b7-0x80000000090c0d00-0x2047000,v1,thead/c900-legacy,core
 0x67e-0x80000000db0000[89]0-0x[[:xdigit:]]+,v1,starfive/dubhe-80,core
 0x31e-0x8000000000008a45-0x[[:xdigit:]]+,v1,andes/ax45,core
-- 
2.47.3

